ci/build-appimage.sh

```diff
@@ -45,6 +45,8 @@ export PIP_WORKDIR="$REPO_ROOT"
 export OUTPUT=appimagelint-x86_64.AppImage
 export VERSION="$SETUPPY_VERSION-git$COMMIT"
 
+mkdir -p AppDir/usr/share/metainfo/ && cp "$REPO_ROOT"/resources/appimagelint.appdata.xml AppDir/usr/share/metainfo/
```

Comment:
Not required for now, but do you know `install`? You could've just `install .../...xml AppDir/usr/share/metainfo`, it'll create all the intermediate directories, you can also spare some `chmod/chown` stuff by making use of optional parameters.
Also `install` can serve as a replacement for `mkdir` using `install -d my/dir`, also allows for changing owners or permissions with the same flags a normal call supports.

Reply:
I like simple tools that everyone understands. But yes, I should learn how to use `install`.

Reply:
`install` is awesome, as it saves you typing work. I thought *you* might be really interested in that therefore. It's part of the GNU coreutils, so it's available nearly anywhere.